testsuite/test_mof_compiler.py

```diff
@@ -1427,35 +1451,27 @@ def test_mof_schema_roundtrip(self):
 
         repo2 = self.mofcomp2.handle
 
-        print('Start recompile file= %s' % mofout_filename)
-
+        # print('Start recompile file= %s' % mofout_filename)
```

Comment:
My only comment here is that this is a long delay for a single test.  If you are watching you begin to wonder. The start recompile gave a clue that it was really running

Reply:
I think it is more useful to have a clean output. py.test shows a dot for each test, and I don't think there is anything wrong is the next dot takes a minute or so to appear.